lib/features/reps_myths/reps_myths.install

```diff
@@ -26,3 +26,27 @@ function reps_myths_disable() {
 function reps_myths_install() {
   _reps_myths_create_listing_page();
 }
+
+/**
+ * Copy post date into field_reps_core_date.
+ */
+function reps_myths_update_7001() {
+  // Revert field instance to be able to copy data in field_reps_core_date.
+  features_revert(array(
+    'reps_myths' => array(
+      'field_instance',
+    ),
+  ));
+  // Selecting nid of type reps_eu_myth.
+  $query = db_select('node', 'n')
+    ->fields('n', array('nid'))
+    ->condition('type', 'reps_eu_myth')
+    ->execute();
+  $results = $query->fetchCol();
+  // Update field_reps_core_date value.
+  foreach ($results as $nid) {
+    $node = node_load($nid);
```

Comment:
instead of loading the nodes 1 by 1 you'd better use node_load_multiple then loop through it